docs/guide/apollo/mutations.md

```diff
@@ -4,7 +4,7 @@ Mutations are queries that change your data state on your apollo server.
 
 Use `this.$apollo.mutate()` to send a GraphQL mutation.
 
-For more info, visit the [apollo doc](https://www.apollographql.com/docs/react/reference/index.html#ApolloClient\.mutate). There is a mutation-focused [example app](https://github.com/Akryum/vue-apollo-todos) you can look at.
+For more info, visit the [apollo doc](https://www.apollographql.com/docs/react/essentials/mutations.html#render-prop). There is a mutation-focused [example app](https://github.com/Akryum/vue-apollo-todos) you can look at.
```

Comment:
If think the right link is https://www.apollographql.com/docs/react/api/apollo-client.html#ApolloClient.mutate

Reply:
Oops, you're right, I just fixed that.